apps/music/js/endpoint.js

```diff
@@ -384,6 +384,12 @@ function setSongRating(rating, filePath) {
 function search(key, query) {
   return new Promise((resolve) => {
     var results = [];
+
+    if (!query) {
```

Comment:
Let's move this to `js/db.js`. My goal is to gradually move as much of the DB-related `endpoint.js` logic into `db.js`.